### PR TITLE
Add Covalent Documentation

### PR DESCRIPTION
**Pull request by themacexpert**
This PR includes:
- Documentation and tutorials for Covalent
- Images and Code Snippets
- Creates new Indexers Folder under Integrations and Moves the Graph to that Location along with Covalent.

### CODE REVIEW

**Comment by eshaben** [comment]
I didn't forget about this! I will review first thing in the AM!

File: integrations/indexers/covalent.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+-u {YOUR API KEY HERE}:
+```
+!!! note
+    The colon `:` after the API key is important because otherwise you will be prompted for a password (which is not needed).
+
+
+The Covalent API will return a list of token holders for the ERTH token. Unless you already owned some ERTH tokens, your address will be missing from that list. Head over to the [Moonbase Alpha ERC-20 Faucet](https://moonbase-minterc20.netlify.app/) to generate some ERTH tokens for yourself. Now repeat the same Covalent API request as above. You should now see your address in the list of token holders for the ERTH token.
+
+## Javascript Examples
+Copy and paste the below code block into your preferred environment, or [JSFiddle](https://jsfiddle.net/). After setting the API key, set the address constant. Remember our chain ID is ‘1287’ for Moonbase Alpha.
```
Comment (eshaben) [inline integrations/indexers/covalent.md:79]:
```suggestion
Copy and paste the below code block into your preferred environment, or [JSFiddle](https://jsfiddle.net/). After setting the API key, set the address constant. Remember our chain ID is `1287` for Moonbase Alpha.
```

File: integrations/indexers/covalent.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+Covalent doesn’t have an official API wrapper. To query the API directly you will have to use the Python [requests library](https://pypi.org/project/requests/). Install requests into your environment from the command line with `pip install requests`. Then import it and use it in your code. Use the HTTP verbs get methods to return the information from the API. Copy and paste the below code block into your preferred environment and run it. 
+
+```python
+import requests
+
+def fetch_wallet_balance(address):
+	api_url = 'https://api.covalenthq.com'
+    endpoint = f'/v1/1287/address/{address}/balances_v2/'
+    url = api_url + endpoint
+    r = requests.get(url, auth=('YOUR API KEY HERE',''))
```
Comment (eshaben) [inline integrations/indexers/covalent.md:146]: what is the second parameter for? in `auth=('YOUR API KEY HERE','')`

**Review by eshaben — APPROVED** [review]
Looks good! ✨  Just one little suggestion and a question re: parameters

File: integrations/indexers/covalent.md (hunk 2 above)
Comment (themacexpert) [reply integrations/indexers/covalent.md:146]: It's actually empty! But I can clarify that in the text.

File: integrations/indexers/covalent.md (hunk 1 above)
Comment (themacexpert) [reply integrations/indexers/covalent.md:79]: Thank you! will fix that.

**Pull request merged**